enterprise/models.py

```diff
@@ -1602,7 +1602,7 @@ def safe_logo_url(self):
 
         # AWS S3 storage is used in stage/production environments but file system
         # storage is used in devstack environment
-        if settings.DEFAULT_FILE_STORAGE == 'storages.backends.s3boto.S3BotoStorage':
+        if settings.DEFAULT_FILE_STORAGE == 'storages.backends.s3boto3.S3Boto3Storage':
```

Comment:
Long term, we should find another way to flag on whether we are in a local vs. stage/prod environment than relying on `settings.DEFAULT_FILE_STORAGE` directly.